packages/components/stencil.config.ts

```diff
@@ -10,6 +10,10 @@ export const config: Config = {
   buildDist: true,
   sourceMap: false,
   validatePrimaryPackageOutputTarget: true,
+  hydratedFlag: {
+    name: 'data-hydrated',
```

Comment:
We could think about using our prefix here as well to make it clear where that comes from. Browsers are following the convention that dashed attribute names are user-land (except for aria-*) so it should be safe to use.

```suggestion
    name: 'post-hydrated',
```

Reply:
If we do this, we should align it on the main also.
Do you want me to do so?

Reply:
Won't do, because the flag is not set by us but by the stencil logic implemented in the components.